### ISSUE TITLE
All tests and CI broken by converting Virtualmin github account to an organization

### ISSUE THREAD

**Issue by swelljoe** [virtualmin/Virtualmin-Config#9]
Since the conversion of Virtualmin account to a github organization all Travis CI connections have stopped working, so tests, coverage reports, etc. are all in a state of limbo (they just aren't running so we see old statuses from months ago). I dunno how to fix it without manually recreating all of the connections, which I simply don't have time for right now. I regret caving to peer pressure on this decision.

**Comment by swelljoe** [comment]
Actually, it's worse than that. I can't even log back into the Virtualmin Travis CI account, since it logs in via github and there is no longer a github virtualmin account. So...it's just broken forever, I guess?

**Comment by swelljoe** [comment]
It also broke automatic mirroring to gitlab, and it looks like it'll have to be resolved on a per-repo basis, as well. Not super important, but that was our repo backup, independent of github, so it needs to get sorted at some point.

**Comment by iliaross** [comment]
> Not super important, but that was our repo backup, independent of github, so it needs to get sorted at some point.

I didn't know we're doing it. It is a great idea, and I think, it's important!

**Comment by swelljoe** [comment]
I'd originally setup a gogs install for the purpose, but gitlab was free even for private repos and already had github syncing built-in, so it proved faster and cheaper. But, now I have to do it all over again. Oh, well. That's the one big pain of having 30+ repos instead of just one big one. Setting up this kind of thing is super tedious...probably scriptable, but it's one of those things that'd probably take a long to script as it would to do it in the GUI and it should be a one-time thing.

**Comment by iliaross** [comment]
How about just adding hooks on `.git/hooks` side and automatically upload after a commit or a push?

**Comment by swelljoe** [comment]
No. That distributes the problem. I think just having it auto-synced by gitlab is good and simple, it just takes time to set it all back up again for every repo.

**Comment by iliaross** [comment]
>  I think just having it auto-synced by gitlab is good and simple

Okay, great! Good to know!

**Comment by chris001** [comment]
@swelljoe  
Someone had the same issue in 2015, and posted on stack overflow.  
Someone else who also had the same issue, answered them with the solution, and here's the post!

https://stackoverflow.com/questions/29658536/conversion-to-github-organization-breaks-travis-ci-pro-setup

**Comment by swelljoe** [comment]
Yeah, I found that, but don't have time to work on it currently. And, we're not paying for Travis CI Pro, so I'm not sure how eager they'll be to help. It is likely to be less time than the manual process of re-adding all of the repos that have CI configs, but still will take at least a couple/few hours to create new user, setup API access, contact support, etc.. Regardless, it'll just have to be broken for a while, as I have negative free time for the foreseeable future.

**Comment by chris001** [comment]
Travis will help fix this kerfluffle, even when you're using the free version of Travis.  Especiially since you're GPL/open source!
